src/main/java/org/javacord/bot/commands/DocsCommand.java

```diff
@@ -39,14 +39,17 @@ public void onCommand(TextChannel channel, String[] args) {
                         .setThumbnail(getClass().getClassLoader().getResourceAsStream("javacord3_icon.png"), "png")
                         .setColor(Constants.JAVACORD_ORANGE);
                 if (args[0].matches("(classes|class|c)")) { // Search for a class
-                    String searchString = String.join(" ", Arrays.copyOfRange(args, 1, args.length));
-                    populateClasses(channel.getApi(), embed, searchString);
+                    boolean searchAll = args[1].matches("(all|a)");
```

Comment:
Actually, do you think allowing to search all makes sense?
If so, it would be nice if that would also search the core-docs, not only the api-docs.

Reply:
included.